core/src/test/java/org/elasticsearch/search/suggest/SuggestSearchIT.java

```diff
@@ -1108,6 +1119,231 @@ public void testSuggestWithManyCandidates() throws InterruptedException, Executi
         // assertThat(total, lessThan(1000L)); // Takes many seconds without fix - just for debugging
     }
 
+    @Override
+    protected Collection<Class<? extends Plugin>> nodePlugins() {
+        return Collections.singleton(DummyTemplatePlugin.class);
+    }
+
+    public static class DummyTemplatePlugin extends Plugin implements ScriptPlugin {
```

Comment:
Have you seen MockScriptPlugin? It probably isn't helpful here but might be a thing to think about. Also, javadocs would help for these I think.